FitnessAPI/database/logEntrySchema.js

```diff
@@ -0,0 +1,7 @@
+let mongoose = require("mongoose");
+
+module.exports = new mongoose.Schema({
```

Comment:
Det skal jo være når brugeren trykker "complete", så laver den et log entry i DB med tidspunktet det pågældende program blev completed. Fancy beskeder og lignede er noget Frostenden står for.